source/state_representation/include/state_representation/exceptions/InvalidStateVariableException.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <exception>
+#include <iostream>
+
+namespace state_representation::exceptions {
+class InvalidVariableStateException : public std::runtime_error {
```

Comment:
Filename is good but not class name :smile:

Reply:
yes yes, already fixed. Sorry